### DISCUSSION TITLE
Github Actions doesn't GET url

### DISCUSSION THREAD

**Discussion by TianJagodic** [community/community#25366] [Actions]
<p>Hello,</p>
<p>I am trying to connect to a URL when a push is made. The problem is that the action just keeps running and does not connect to the server.</p>
<p>I am new to Github actions, so here is my code:</p>
<pre><code class="lang-auto">name: SendURL

on:
  push:
    branches: [ master ]
jobs:
  deployment:
    runs-on: ubuntu-latest
    steps:
    - name: Deploy Stage
      uses: fjogeleit/http-request-action@master
      with:
        url: 'http://193.138.42.XXX:41101/?OauthID=someID&amp;project=someProject'
        method: 'GET'
</code></pre>
<p>The XXX in the IP is just to cover my actual IP.</p>
<p>The only this I need is that the action contacts the server and gives it the OauthID and project name.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>For a possible bug in the action it might be better to raise an issue in the action repository, but if you just need to send a GET request to that URL you could simply use <code>curl</code>, like so:</p>
<pre><code class="lang-yaml">    steps:
      - name: Deploy Stage
        run: curl http://193.138.42.XXX:41101/?OauthID=someID&amp;project=someProject
</code></pre>

**Comment by Yanjingzhu** [comment]
<p>As your job runs-on ubuntu-latest runner, it is a GitHub Hosted runner which installed on Azure VMs . Is the url accessible through the internet?  If not, your server could not contact with your server. You could consider to use a <a href="https://help.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners" rel="nofollow noopener">self-hosted runner</a> if possible.</p>
